server/views/pages/prisonerList/partials/searchTable.njk

```diff
@@ -0,0 +1,79 @@
+{# TODO: Logic for the problemRetrievingData #}
+{% if not ciagPrisonerList.problemRetrievingData %}
```

Comment:
Preempting the `problemRetrievingData` logic here, maybe abit early. 

Just a placeholder name for now.